olive/model/hf_utils.py

```diff
@@ -25,6 +27,137 @@ class HFComponent(ConfigBase):
     dummy_inputs_func: Union[str, Callable]
 
 
+class HFModelLoadingArgs(ConfigBase):
```

Comment:
kwargs?

Reply:
Do you mean something like `HFModelLoadingKwargs`?

Reply:
yeah, anyway you can ignore the comment

Reply:
Okay, thanks! I will keep it as it is for now. Since kwargs might give the idea that we can pass anything here. But here we need to pass `extra_args` dict if they want to pass more kwargs.